### PR TITLE
LINK-1388 | Updated Terms of Use -page information

### PR DESCRIPTION
**Pull request by minnas**
## Description :sparkles:

This PR updates Term Of Use -page texts

## Issues :bug:

**[LINK-1388](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1388):**

## Testing :alembic:

Tested locally

[LINK-1388]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1388?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx [hunk 1]
```diff
@@ -64,27 +141,106 @@ const TermsOfUsePage: React.FC = () => {
...
+              lakisääteisen velvoitteen noudattamiseksi.
+            </p>
+            <p>
+              Tietoja säilytetään pääsääntöisesti 5 vuotta, mutta pyynnöstä ne
+              poistetaan nopeammin.
+            </p>
+            <p>
+              Helsingin kaupungin käyttöoikeuksien tietosuojaseloste
+              {/*eslint-disable*/}
+              <ExternalLink href="https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Keha/Sahkoisten%20asiointipalveluiden%20rekisteri.pdf">
```
Comment (mikkojamG) [inline src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx:211]: Screenshot 2023-08-25 at 14 31 01

Looks like links are missing space between the word before. Also I think the second "Tietosuojaseloste" can be removed here 🤔

File: src/domain/help/pages/termsOfUsePage/termsOfUsePage.module.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.termOfUse {
+  list-style-type: none;
+}
+.termOfUse li:before {
```
Comment (mikkojamG) [inline src/domain/help/pages/termsOfUsePage/termsOfUsePage.module.scss:4]: Screenshot 2023-08-25 at 14 31 21

Looks like the list items could use `var(--color-black-70);`. There is a bit of color mismatch with the body text 🤔

File: src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx [hunk 3]
```diff
@@ -13,20 +14,96 @@ const TermsOfUsePage: React.FC = () => {
       case 'en':
         return (
           <>
-            <h1>Terms of use</h1>
+            <h1>Data protection and terms of use</h1>
```
Comment (mikkojamG) [inline src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx:17]: Looks like there are unit tests for checking these headings in https://github.com/City-of-Helsinki/linkedcomponents-ui/blob/develop/src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx#L525. Should change the titles here 🙂

**Review by mikkojamG — CHANGES_REQUESTED** [review]
Good job! 👍 Left couple of comments:

File: src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx (hunk 1 above)
Comment (minnas) [reply src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx]: Added space before links

File: src/domain/help/pages/termsOfUsePage/termsOfUsePage.module.scss (hunk 2 above)
Comment (minnas) [reply src/domain/help/pages/termsOfUsePage/termsOfUsePage.module.scss:5]: Updated correct colour

File: src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx (hunk 3 above)
Comment (minnas) [reply src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx:17]: Updated test datas

File: src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx [hunk 4]
```diff
@@ -80,15 +79,13 @@ const testHelpPage = async (
   route: ROUTES,
   expectedValues: PageValues
 ) => {
-  const { description, expectedRoute, keywords, pageTitle, title } =
+  const { description, expectedRoute, keywords, /*pageTitle,*/ title } =
```
Comment (mikkojamG) [inline src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx:82]: This commented code can be removed 🙂

**Review by mikkojamG — APPROVED** [review]
One nitpicky fix, I think after that it's good to go 👍

**Review by jorilindell — APPROVED** [review]
Looks good. Commented code could be removed as @mikkojamG already mentioned.

**Pull request merged**